src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -57,6 +57,10 @@ public Http2Connection(HttpConnectionPool pool, SslStream stream)
             _nextStream = 1;
         }
 
+        public bool UseCookie => _pool.Settings._useCookies;
+
+        public CookieContainer GetCookieContainer => _pool.Settings._cookieContainer;
```

Comment:
if this is a property should this just be CookieContainer?

Reply:
I agree. This should be named "CookieContainer"

Reply:
Fixed.